README.md

```diff
@@ -1,18 +1,44 @@
-# AFrame & Redux - What This Is
+# A-Frame + Redux
 
-At [Olio Apps](http://www.olioapps.com/), we are applying software engineering idioms from React and React Native development ecosystems to WebVR development.  
+At [Olio Apps](http://www.olioapps.com/), we are applying tools and practices from our experience creating React and React Native applications to engineering VR software for the web.  
 
-This project is our effort to make a development platform that can be used to create aframe components and subclasses which makes it easy to connect your VR components to a redux store, for these reasons:
+## Installation 
 
-- to facilitate the use of data driven logic.
-- to be able to manage state the same way as other front end javascript/redux applications, and also to be able to think about the components in a familiar React-esque way.
-- to facililate easier onboarding for new and more experienced developers for rapid development and component creation.
+### npm
```

Comment:
i think that the installation instructions presented here are relevant to people who want to contribute to the source code of the redux framework files. while that is good, at this stage i think we should focus more on showing people how to use the redux-connected component.

lets remove the `npm` section for now, and just leave in the `cdn` bit, and reintroduce local development/contribution after we properly release aframe-typescript-toolkit.